README.md

```diff
@@ -1,60 +1,60 @@
 
-## Import the example application
+Please install Arm Mbed CLI.
 
-From the command-line, import the example:
+Clone this repository on your system and change the current directory to where the project was cloned.
 
-```
-mbed import mbed-os-example-sys-info
-cd mbed-os-example-sys-info
-```
+## Application functionality
 
-### Now compile
+The `main()` function outputs on the serial interface information about the hardware (CPU, ROM and RAM), the firmware (Version), as well as the compiler used to build the binary.
 
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the ARM Compiler 5:
+## Building and Running
 
+1. Connect a USB cable between the USB port on the target and the host computer.
+2. Run the following command to build the example project and program the microcontroller flash memory:
+```bash
+$ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash
 ```
-mbed compile -m K64F -t ARM
-```
+Your PC may take a few minutes to compile your code.
 
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-sys-info.bin` and can alternatively be manually copied to the target which gets mounted on the host computer via USB.
 
-```
-[snip]
-+------------------+-------+-------+------+
-| Module           | .text | .data | .bss |
-+------------------+-------+-------+------+
-| [lib]\c_w.l      | 11473 |    16 |  348 |
-| [lib]\cpprt_w.l  |    36 |     0 |    0 |
-| [lib]\fz_wm.l    |    18 |     0 |    0 |
-| [lib]\m_wm.l     |    48 |     0 |    0 |
-| anon$$obj.o      |    32 |     0 | 1024 |
-| main.o           |   136 |     0 |    0 |
-| mbed-os\drivers  |   130 |     0 |    0 |
-| mbed-os\features |   132 |     0 |  304 |
-| mbed-os\hal      |  1660 |    30 |   64 |
-| mbed-os\platform |  3465 |   104 |  604 |
-| mbed-os\rtos     | 12926 |  2310 | 4592 |
-| mbed-os\targets  |  9193 |   104 |  324 |
-| Subtotals        | 39249 |  2564 | 7260 |
-+------------------+-------+-------+------+
-Total Static RAM memory (data + bss): 9824 bytes
-Total Flash memory (text + data): 41813 bytes
-
-Image: .\BUILD\K64F\ARM\mbed-os-example-sys-info.bin
-```
+Depending on the target, the example project can be built with GCC_ARM, ARM or IAR toolchain. Run the command below after installing ARM Mbed CLI to determine which toolchain supports your target.
 
-### Program your board
+```bash
+$ mbed compile -S
+```
 
-1. Connect your Mbed device to the computer over USB.
-1. Copy the binary file to the Mbed device.
-1. Press the reset button to start the program.
+## Expected output 
+The LED blinks and the serial terminal shows an output similar to the following: 
+``` 
+--- Terminal on /dev/ttyACM0 - 9600,8,N,1 --- 
+Mbed OS Version: 999999  
+CPU ID: 0x410fc241  
+Compiler ID: 1  
```

Comment:
It would be worth adding info on what the IDs mean:
(ARM : 1, GCC_ARM : 2, IAR : 3)